.travis.yml

```diff
@@ -7,6 +7,7 @@ android:
   - tools
   - platform-tools
   - android-28
+  - build-tools-29.0.2
```

Comment:
Remove in both places?

Reply:
Travis was failing to build with `Warning: License for package Android SDK Build-Tools 29.0.2 not accepted.`, so I added this dependency here. I think I can remove it from `build.gradle` though.